src/main/java/core/basesyntax/strategy/operations/BalanceOperation.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.strategy.operations;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Transaction;
+import core.basesyntax.strategy.OperationStrategy;
+
+public class BalanceOperation implements OperationStrategy {
```

Comment:
```suggestion
public class BalanceOperationHandler implements OperationHandler {
```
same for all handlers